controller_node:
  ros__parameters:
    goals_list: 
      - 3.0
      - 4.0
      - 5.0
      - 8.0
    velocity_vector: 
      - 0.5
      - 0.0
      - 0.0
      - 0.0
      - 0.0
      - 0.5